docs/src/main/paradox/scala/http/migration-guide/index.md

```diff
@@ -7,6 +7,6 @@
 
 * [migration-from-spray](migration-from-spray.md)
 * [migration-guide-2.4.x-10.0.x](migration-guide-2.4.x-10.0.x.md)
-* [migration-within-10.0.x-Api-May-Change](migration-within-10.0.x-Api-May-Change.md)
+* [migration-guide-10.0.x](migration-guide-10.0.x.md)
```

Comment:
Yes, let's keep it simple and collect all those information in one place.

In the middle-term, we'll try not to lose compatibility with 2.4 so it should at least be possible to still use Akka 2.4 for a while regardless of whether we compile our binaries against 2.5 or 2.4.